src/AutoMapper/Mappers/MapperRegistry.cs

```diff
@@ -4,7 +4,7 @@ namespace AutoMapper.Mappers
 {
     public static class MapperRegistry
     {
-        private static readonly IObjectMapper[] _initialMappers =
+        public static IObjectMapper[] Mappers() => new IObjectMapper[]
         {
```

Comment:
Ha! I was trying out the Review feature in GitHub. Honestly don't understand the point of it.